.github/scripts/tests/test_themes.py

```diff
@@ -77,3 +80,11 @@ def verify_theme_data(theme_name: str) -> None:
     s.add_frame(approvaltests.utils.to_json(theme.data()))
 
     verify(s)
+
+
+@contextlib.contextmanager
+def silence_stdout():
```

Comment:
Hi Andy, one of the checks in the python GitHub Actions is asking for a type hint to be added here, on the return value of the new function:

https://github.com/obsidian-community/obsidian-hub/actions/runs/3577679728/jobs/6017487017#step:6:9

I recall I struggled a bit getting PyCharm to point out missing type hints - I probably have some notes.
Until then I ran this script if I wanted to check them locally:
`.github/scripts/check_type_hints.sh`

Reply:
Thank you - I see the tests are passing now.